north/src/runtime/mount.rs

```diff
@@ -431,3 +437,24 @@ async fn wait_for_file_deleted(path: &Path, timeout: time::Duration) -> Result<(
         .and_then(|r| r.map_err(Error::JoinError))
         .and_then(|r| r)
 }
+
+fn is_version_supported(manifest: &Manifest) -> bool {
+    const SUPPORTED_MIN_VERSION: &Version = &VERSION_1_0_0;
+    const KNOWN_MAX_VERSION: &Version = &VERSION_1_0_0;
+    const DEFAULT_VERSION: &Version = &VERSION_1_0_0;
+    let manifest_version = manifest
+        .manifest_version
+        .as_ref()
+        .unwrap_or(DEFAULT_VERSION);
+
+    // compare major versions
+    if manifest_version.0.major < SUPPORTED_MIN_VERSION.0.major {
```

Comment:
The manifest version change should not be part of this PR. We should keep it separate so that we can further refine the concept since we already found some problems with it yesterday.